tracer/test/Datadog.Trace.Security.IntegrationTests/Rcm/AspNetCore5RcmClient.cs

```diff
@@ -0,0 +1,67 @@
+// <copyright file="AspNetCore5RcmClient.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NETCOREAPP3_0_OR_GREATER
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading.Tasks;
+using Datadog.Trace.AppSec;
+using Datadog.Trace.AppSec.Rcm.Models.AsmFeatures;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.RemoteConfigurationManagement;
+using Datadog.Trace.RemoteConfigurationManagement.Protocol;
+using Datadog.Trace.TestHelpers;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.Security.IntegrationTests.Rcm
+{
+    public class AspNetCore5RcmClient : RcmBase
```

Comment:
I dont have a strong opinion on this and I'm not sure where they should be otherwise, but are these tests  specifically related to `Security` ? other teams might be interested in adding or editing them later, so I was wondering if they should be in our security integration tests and samples 🤔 and maybe instead in a the general integration tests `Datadog.Trace.IntegrationTests`, but could be something to keep in mind for later maybe :)

Reply:
It's not specific to security, but we're the only team using it for the moment.

It probably would better to move it into the core tests, but all the infrastructure for testing remote config is here, so I'd prefer to leave it here for the moment. We can refactor later if we start to find many tests related to core are placed here.